controller/src/main/java/org/jboss/as/controller/AbstractControllerService.java

```diff
@@ -116,10 +117,16 @@ private static int getBootStackSize() {
         }
     }
 
+    /** Capability in-vm users of the controller use to create clients */
     protected static final RuntimeCapability<Void> CLIENT_FACTORY_CAPABILITY =
             RuntimeCapability.Builder.of("org.wildfly.managment.model-controller-client-factory", ModelControllerClientFactory.class)
             .build();
 
+    /** Capability in-vm users of the controller use to register notification handlers */
+    protected static final RuntimeCapability<Void> NOTIFICATION_REGISTRY_CAPABILITY =
+            RuntimeCapability.Builder.of("org.wildfly.managment.notification-handler-registry", NotificationHandlerRegistry.class)
```

Comment:
@jmesnil here's the PR for the ones I've been doing this month:

https://github.com/bstansberry/wildfly-capabilities/pull/3